animal_adoption/routes.py

```diff
@@ -42,8 +46,10 @@ def login():
 
     result = User.authenticate_user(username=username, password=password)
     if result:
-        access_token = create_access_token(identity=username)
-        return jsonify(access_token=access_token), 200
+        access_token = create_access_token(identity=User.get_id_by_username(username))
+        response = make_response(redirect('/', 200))
```

Comment:
I think  this line(#50) returns Redirect response and our front-end needs JSON response(it cannot handle a redirect response)
So how about using jsonify like? 
```
response = jsonify(message='Login Succeeded!')
```
jsonify also returns Flask response object. So we can pass it as it is to `set_access_cookies`'s first parameter.